import-export-cli/integration/testdata/TestArtifactDirectory/DevFirstUpdatedSampleCaseArtifacts/customAddLogMessage.j2

```diff
@@ -0,0 +1,4 @@
+<log level="custom">
+     <property name="{{propertyName}}" value="{{propertyValue}}"/>
+     <property name="VALUE is " value="456"/>
```

Comment:
The j2 we are uploading doesn't have this `<property name="VALUE is " value="456"/>` right? Shall we remove this? Or is there a use for this second property? On that note, do we do a content check on the exported policies?

Reply:
Since this is sequence update test case, in the first step we push the sequence [1]. Then we reimport the API with the above-pointed sequence [2] while updating the API as the second step. Then, we export the API and compare the exported sequence with the one we updated in the second step. So, we use this second property, hence we cannot remove it.

[1] https://github.com/wso2/product-apim-tooling/pull/891/files#diff-da64a758313dbf33e467d6579861e61ce68e5b535ffa436d605f35afb393e04f
[2] https://github.com/wso2/product-apim-tooling/pull/891/files#diff-c26c9ec7c553c52743711d068999e18ed836d2afacdcc60da797ff570a12eb7f